tests/templates/test_issue#12_intermediate.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'test_issue#12_base.txt' %}
+
+{% block block1 %}
+{{ block.super }}
```

Comment:
Isn't this not valid in Django templating engine? Are you using Jinja2?

Reply:
@clokep  I'm using Django. Are you referring to the {{block.super}}? Or to the additional block I created afterwards? 

Well, I can see the block names (in context), so I'm guessing it's valid. I guess I can try the same in a webpage and then have it rendered by Django

Reply:
I'm referring the `{{ block.super }}` call.

Reply:
You're right though, I'm just mixing up the syntax between the two. `block.super` is a Django templating engine thing: https://docs.djangoproject.com/en/1.11/ref/templates/language/